internal/cmd/connector_test.go

```diff
@@ -72,6 +75,15 @@ func TestConnector_Run(t *testing.T) {
 	ctx := context.Background()
 	runAndWait(ctx, t, srv.Run)
 
+	kubeconfig := path.Join(dir, "kubeconfig")
```

Comment:
This is probably the only downside to using the `client-go` loading: a bit more pain in writing tests since we have to generate a full KubeConfig. It's not too bad